build-image/first-boot.bash

```diff
@@ -111,7 +111,7 @@ elif grep -qs "openHABian" /etc/wpa_supplicant/wpa_supplicant.conf && ! grep -qs
   else
     echo "OK"
   fi
-  if tryUntil "ping -c1 8.8.8.8 &> /dev/null || curl --silent --head http://www.openhab.org/docs |& grep -qs 'HTTP/1.1 200 OK'" 5 1; then
+  if tryUntil "curl --silent --head http://www.example.com/ |& grep -qs 'HTTP/1.1 200 OK'" 5 1; then
```

Comment:
You must not change this.
Actually with the early networking stages there can be a case where *any* URL with nothing but a hostname resolves to the *same* IP address and the curl command actually succeeds.
That's why I added a string to the URL used in this test

Reply:
Yeah, I was having issues with this during my initial testing and forgot to change it back, will do.

Reply:
Make sure you don't get victim to that apparently working inet link like I got first time.